server/templates/simulate.html.tmpl

```diff
@@ -0,0 +1,62 @@
+{{/* templatetree:extends base-details.html.tmpl */}}
+{{define "title"}}{{.PullRequest.GetBase.GetRepo.GetFullName}}#{{.PullRequest.GetNumber}} - Details | PolicyBot{{end}}
+
+{{define "scripts"}}
+  <script defer src="{{ resource "js/filter.js" }}"></script>
+  <script defer src="{{ resource "js/prefill.js" }}"></script>
+{{end}}
+
+{{define "body-class"}}bg-light-gray5 text-dark-gray1 flex flex-col h-screen{{end}}
+
+{{define "top"}}
+  <header class="w-full tripart p-4 bg-white shadow-sm z-10 relative">
+    <a href="{{.PolicyURL}}" title="View the policy definition on GitHub"
+       class="px-2 py-1 text-xs text-dark-gray3 bg-light-gray3 border border-light-gray2 rounded-sm truncate max-w-full hover:bg-light-gray2 no-underline hover:no-underline">
+      {{.PullRequest.GetBase.GetRepo.GetFullName}}: {{.PullRequest.GetBase.GetRef}}
+    </a>
+    <h1 class="text-xl font-normal tracking-tight text-center">
+      <a href="{{.PullRequest.GetHTMLURL}}" title="View the pull request on GitHub" class="text-blue3 hover:text-blue4 no-underline">
+        #{{.PullRequest.GetNumber}}</a>:
+      {{.PullRequest.GetTitle}}
+    </h1>
+    <span class="text-xs text-dark-gray3 truncate max-w-full">
+      {{.User}}
+    </span>
+  </header>
+  <div class="bg-light-gray3 border-b border-gray3">
+    <div class="pl-5 pt-5">
+      <form class="w-full max-w-md" method="get">
+        <div class="md:flex md:items-center mb-6">
```

Comment:
As a general rule, I've tried to avoid the media query based classes (`md:` prefix) in favor of things that naturally scale and adjust as the screen size changes. I'd probably see how this works with the un-prefixed classes and then adjust from there.